package.json

```diff
@@ -36,6 +36,8 @@
     "@luma.gl/constants": "9.0.28",
     "@luma.gl/gltf": "9.0.28",
     "@emotion/utils": "^1.1.0",
-    "@emotion/sheet": "^1.1.0"
+    "@emotion/sheet": "^1.1.0",
+    "htmlparser2": "9.1.0",
```

Comment:
NOTE: This a version force for a couple of libraries used in @hylo/shared for htmlToText. The dependency is shared was lower than this, and not necessarily compatible at least in smear terms (I think it requires 7.x of htmlparser2). Nothing seems to be broken, and after to way too much time into I cam to the conclusion that there was no better way.